src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
+import { User } from './types/User';
+import { Todos } from './types/Todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getTodosId(todoId: Todos[]) {
+  const maxTodoId = Math.max(...todoId.map(todo => todo.id));
+
+  return maxTodoId + 1;
+}
+
+const initialTodos: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
```

Comment:
Here you can use a variable like this: ``const DEFAULT_USER_ID = 0;`` to avoid magic values.
```suggestion
  const [userId, setUserId] = useState(DEFAULT_USER_ID );
```